tac/agents/v1/agent.py

```diff
@@ -134,3 +135,10 @@ def react(self) -> None:
 
         :return: None
         """
+
+    @abstractmethod
+    def update(self) -> None:
+        """Update the current state of the agent.
```

Comment:
Note, this is ok for now, but going forward we need to introduce more explicit language around what is the agent state vs what is game specific 'AgentState'. The former is a much wider domain than the latter.

Reply:
Agreed!